src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -56,11 +56,17 @@ class Serializer implements SerializerInterface, NormalizerInterface, Denormaliz
 
     /**
      * @var array
+     *
+     * @deprecated since 3.1 will be removed in 4.0
+     * @see https://github.com/symfony/symfony/pull/17140
```

Comment:
I think we should remove the line (and the same below): git blame gives the history, that's good enough (and matches the current way of doing things AFAIK)